src/dotnet/CarbonAware.WebApi/Models/SerializableEmissionsForecast.cs

```diff
@@ -0,0 +1,44 @@
+﻿using CarbonAware.Model;
+using System.Text.Json.Serialization;
+
+namespace CarbonAware.WebApi.Models;
+
+[Serializable]
+public record SerializableEmissionsForecast
+{
+    [JsonPropertyName("generatedAt")]
+    public DateTimeOffset GeneratedAt { get; set; }
+
+    [JsonPropertyName("location")]
+    public string Location { get; set; } = string.Empty;
+
+    
+    [JsonPropertyName("windowSize")]
+    public int WindowSize { get; set; }
+
+    [JsonPropertyName("notBefore")]
+    public DateTimeOffset NotBefore { get; set; }
+
+    [JsonPropertyName("notAfter")]
+    public DateTimeOffset NotAfter { get; set; }
+
+    [JsonPropertyName("optimalDataPoint")]
+    public EmissionsData? OptimalDataPoint { get; set; }
+
+    [JsonPropertyName("forecastData")]
+    public IEnumerable<EmissionsData>? ForecastData { get; set; }
+
+    public static SerializableEmissionsForecast FromEmissionsForecast(EmissionsForecast emissionsForecast)
```

Comment:
Why is it important to separate SerializableEmissionsForecast and EmissionsForecast if they have the same variables?

Reply:
This is a best practice for separating concerns.  Changes to the underlying model can happen without changing the API contract, and the API contract can change without impact other services using the core models.